README.md

```diff
@@ -35,37 +36,30 @@ var hue = jsHue();
 Then you can discover local bridges:
 
 ```js
-hue.discover(
-    function(bridges) {
-        if(bridges.length === 0) {
-            console.log('No bridges found. :(');
-        }
-        else {
-            bridges.forEach(function(b) {
-                console.log('Bridge found at IP address %s.', b.internalipaddress);
-            });
-        }
-    },
-    function(error) {
-        console.error(error.message);
+hue.discover().then(bridges => {
+    
+    if(bridges.length === 0) {
+        console.log('No bridges found. :(');
     }
-);
+    else {
+        bridges.forEach(function(b) {
+            console.log('Bridge found at IP address %s.', b.internalipaddress);
+        });
+    }
+    
+}
+).catch((err) => {console.log('Error finding bridges', err)});
 ```
 
-jsHue performs requests asynchronously and provides a callback interface. The
-failure callback is called if the XHR request fails, or if there is an error with
-the JSON serialization or deserialization. All API success and error results are
-returned in the success callback data. (This is because some API calls may return
-aggregated API success and error results.)
+jsHue-promise performs requests asynchronously and provides a Promise interface. You can `catch` errors as usual, like in the example below.
 
```

Comment:
Sorry for my absence. We could potentially change the behaviour to check for `failure` hue API responses (in `_requestJson`) and throw an error in the promise, so that all errors are catchable and only `success` objects get returned in a `then`. Would that be preferable or better leave it as it is?